pull_request_template.md

```diff
@@ -1,13 +1,31 @@
+<!-- @formatter:off -->
+
 Why this PR is needed
-----
-Add a sentence or two on the reason this PR exists. Remember that the reviewer likely doesn't have as much context as you. Help them help you.
+---------------------
+<!-- Add a sentence or two on the reason this PR exists. Remember that the reviewer likely doesn't have as much context as you. Help them help you. -->
+
+
 
-Jira ticket reference : [PROJECT-XXX](https://energyhub.atlassian.net/browse/PROJECT-XXX)
+Jira ticket reference: PIT-XXX
 
 What this PR includes
-----
-Brief description of what was done to meet the PR goal.
+---------------------
+<!-- Brief description of what was done to meet the PR goal. -->
+
+
 
 How this change was tested
-----
-Describe testing done on the change, or why this is not applicable.
+--------------------------
+<!-- Describe testing done on the change, or why this is not applicable. -->
+
+
+
+<!--
+Did You Think About
+------------------
+- Monitoring?
+- Metrics and logging to diagnose problems?
+- Documentation?
+-->
```

Comment:
This section is just reminders. Considered making this a visible section, or checkboxes, but that seemed annoying. This way you can just ignore them when filling out the PR template